src/rpc.rs

```diff
@@ -1,28 +1,21 @@
 use std::pin::Pin;
 
-use dcs::mission_server::{Mission, MissionServer};
+use dcs::mission_server::Mission;
+use dcs::streamer_server::Streamer;
 use dcs::*;
 use dcs_module_ipc::IPC;
 use futures::{Stream, StreamExt};
-use tonic::transport::server::Router;
-use tonic::transport::{self, Server};
 use tonic::{Request, Response, Status};
 
 pub mod dcs {
     tonic::include_proto!("dcs");
 }
 
 pub struct RPC {
-    ipc: IPC<Event>,
+    pub ipc: IPC<Event>,
 }
 
 impl RPC {
-    pub fn builder(
-        ipc: IPC<Event>,
-    ) -> Router<MissionServer<RPC>, transport::server::Unimplemented> {
-        Server::builder().add_service(MissionServer::new(RPC { ipc }))
-    }
```

Comment:
A common approach in Rust here would be to use a `new` _constructor_:

```rust
    pub fn new(ipc: IPC<Event>) -> Self {
        RPC { ipc }
    }
```

Reply:
Done